backend/src/User/user.controller.ts

```diff
@@ -25,11 +26,11 @@ class Controller {
 
         // Login user
         this.router.post('/login', async (req: Request, res: Response) => {
-            // If in here, it means the email and password are good, we can log the user
-            const b64auth = (req.headers.authorization || '').split(' ')[1] || '';
-            const [email, password] = Buffer.from(b64auth, 'base64').toString().split(':');
+            const user = auth(req);
+            const name = user ? user.name : undefined;
+            const pass = user ? user.pass : undefined;
```

Comment:
You can use the optional chaining operator ``?.`` to achieve the same behavior.
```suggestion
            const name = user?.name;
            const pass = user?.pass;
```